java/sage/MediaServer.java

```diff
@@ -560,6 +594,10 @@ protected void sizeFile() throws java.io.IOException
           availSize = xcoder.getVirtualTranscodeSize();
           totalSize = xcoder.isTranscodeDone() ? availSize : getLargeFileSize(currFile.toString());
         }
+        if (remuxer != null)
+        {
+          availSize = totalSize = remuxer.getFileSize();
```

Comment:
Fixed. There is a way for the network encoder to let the remuxer know that it needs to create a circular file.